index.bs

```diff
@@ -3070,10 +3070,101 @@ Issue: Fill in this section.
 
 # Privacy considerations # {#privacy-considerations}
 
+## Clearing site data ## {#clearing-site-data}
+
 The [=attribution caches=] contain data about a user's web activity. As such,
 the user agent MAY expose controls that allow the user to delete data from them.
 
-Issue: Discuss general principles behind design decisions for the API.
+## Cross-site information disclosure ## {#cross-site-information-disclosure}
```

Comment:
Do sections like this typically get marked non-normative? E.g. I was looking at WebBluetooth as a spot check (https://webbluetoothcg.github.io/web-bluetooth/#privacy) and quite a bit if marked non-normative.

Reply:
It seems reasonable to do this.